.github/workflows/playwright.yml

```diff
@@ -105,18 +90,16 @@ jobs:
         run: |
           npx playwright install chromium
       - name: Run Playwright tests
-        if: ${{matrix.testSuite !='elements-regression'}}
-        env:
-          TEST_SUITE: "@${{matrix.testSuite}}"
-        run: npm run test:playwright -- --reporter=${{inputs.reporter || 'list,github'}}
+        if: ${{ matrix.shardIndex !='elements-regression' }}
```

Comment:
```suggestion
        if: ${{ matrix.shardIndex != 'elements-regression' }}
```

Reply:
Fixed in https://github.com/elementor/elementor/pull/27617/commits/ba404aa8d49a71f63e0279353aac83b8961b6182